core/src/impl/CMakeLists.txt

```diff
@@ -1,13 +1,15 @@
 
 SET(HEADERS "")
+SET(C_HEADERS "")
 SET(SOURCES "")
 
 FILE(GLOB HEADERS *.hpp)
 FILE(GLOB SOURCES *.cpp)
+FILE(GLOB C_HEADERS *.h)
 
 TRIBITS_ADD_LIBRARY(
     kokkoscore_impl
-    NOINSTALLHEADERS ${HEADERS}
+    NOINSTALLHEADERS ${HEADERS} ${C_HEADERS}
```

Comment:
Is there a good reason for having a separate variable for C headers?  Just curious.

Because if it does not matter which I suspect, you could also have specified multiple blobbing expression which I suspect is more efficient.
```CMake
FILE(GLOB HEADERS "*.hpp" "*.h")
```

Reply:
I'm game to implement it that way, I'm never sure where CMake allows multiple things, so I implemented it the safe way rather than the concise one. I'll push a fix shortly

Reply:
Yep, merged. I don't have the ability to test this one though, I don't know how to run a TRIBITS build. Do they need to be stringified? I have

`FILE(GLOB HEADERS *.hpp *.h)`

Reply:
https://stackoverflow.com/questions/7533502/how-can-i-merge-multiple-lists-of-files-together-with-cmake has it as strings but I think it should be fine as is.